save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -55,4 +58,19 @@ class VulnerabilityController(
     ): Mono<VulnerabilityDto> = blockingToMono {
         vulnerabilityService.getVulnerabilityWithDescriptionByName(name)
     }
+
+    @PostMapping("/save")
+    @Operation(
+        method = "POST",
+        summary = "Save vulnerability.",
+        description = "Save vulnerability.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully saved vulnerability")
+    fun save(
+        @RequestBody vulnerabilityDto: VulnerabilityDto,
+    ): Mono<StringResponse> = blockingToMono {
+        vulnerabilityService.save(vulnerabilityDto)
+    }.map {
+        ResponseEntity.status(HttpStatus.OK).body("Vulnerability was successfully saved")
```

Comment:
```suggestion
        ResponseEntity.ok("Vulnerability was successfully saved")
```

This should be better